tests/unit/test_event.py

```diff
@@ -0,0 +1,130 @@
+import pytest
+from cdf_fabric_replicator.event import EventsReplicator
+from cognite.extractorutils.base import CancellationToken
+from cognite.extractorutils.statestore import LocalStateStore
+from cognite.client.data_classes import EventList, Event
+
+
+@pytest.fixture(scope="session")
+def event_replicator(config):
+    stop_event = CancellationToken()
+    event_replicator = EventsReplicator(metrics=None, stop_event=stop_event)
+    event_replicator.config = config
+    event_replicator.client = event_replicator.config.cognite.get_cognite_client("test_event_replicator")
+    event_replicator.state_store = LocalStateStore(
+        event_replicator.config.extractor.state_store.local.path
+    )
+
+    return event_replicator
+
+@pytest.fixture
+def early_created_time():
+    return 1710503304020
```

Comment:
Can we use this as a constant instead EARLY_CREATED_TIME = 1710503304020

Reply:
Originally thought it would make it clearer to be referred among other fixtures and test methods, but doesn't seem worth the complication. Used a constant instead to simplify the code.